src/main/java/yappingbot/ui/Ui.java

```diff
@@ -0,0 +1,106 @@
+package yappingbot.ui;
+
+import yappingbot.exceptions.YappingBotException;
+
+/**
+ * User-interface class for mainly outputting different texts and decorating them to represent
+ * the bot talking.
+ */
+public class Ui {
+    private static final String PREFIX = "|  ";
+    private static final String PREFIX_EMPTY = "|";
+
+
+    /**
+     * Decorates the given string, to denote that it is the bot's output.
+     *
+     * @param line the message to be decorated.
+     * @return String of the newly decorated message.
+     */
+    private static String quoteSinglelineText(String line) {
+        if (line == null || line.trim().isEmpty()) {
+            return PREFIX_EMPTY + "\n";
+        } else {
+            return PREFIX + line.replaceAll("\n", "") + "\n";
+        }
+    }
+
+    /**
+     * Decorates the given string, to denote that it is the bot's output.
+     * This accepts a StringBuilder for efficiently dealing with multiple lines.
+     *
+     * @param line String to be decorated.
+     * @param sb StringBuilder that the decorated line will be appended to.
+     */
+    protected static void quoteSinglelineText(String line, StringBuilder sb) {
+        if (line.trim().isEmpty()) {
+            sb.append(PREFIX_EMPTY);
+        } else {
+            sb.append(PREFIX);
+            sb.append(line.replaceAll("\n", ""));
+        }
+        sb.append("\n");
+    }
+
+
+    /**
+     * Decorates the given string, to denote that it is the bot's output.
+     * This accepts multiple lines
```

Comment:
Minor nitpick - I think you miss a period here.

Reply:
Ah how did checkstyle miss this...
Good catch!